Documentation/deploy-master.md

```diff
@@ -22,6 +22,8 @@ Place the keys generated previously in the following locations:
 
 [flannel][flannel-docs] provides a key Kubernetes networking capability &mdash; a software-defined overlay network to manage routing of the [Pod][pod-overview] network.
 
+*Note:* If the pod-network is being managed independently of flannel, this step can be skipped. See [kubernetes networking](kubernetes-networking.md) for more detail.
```

Comment:
I'd be great if we could work these into the guide without having to call them out as "notes". I can think on this.